sandbox.html

```diff
@@ -472,9 +473,28 @@
   </div>
 </template>
 
+<template id="common-setCustomValidity" >
+  <fieldset>
+    <legend>setCustomValidity</legend>
+    <label for="customValidityType">Type</label>
+    <select name="customValidityType" id="customValidityType">
+      <option>success</option>
+      <option>info</option>
+      <option>warning</option>
+      <option selected="selected">error</option>
+    </select>
+    <label for="customValidityMessage">Message</label>
+    <input type="text" name="customValidityMessage" id="customValidityMessage">
+    <button type="button" id="setCustomValidityButton" onclick="actionHandlers['setCustomValidity']({
+      type: document.getElementById('customValidityType').value,
+      message: document.getElementById('customValidityMessage').value
+    });">Send</button>
+  </fieldset>
+</template>
+
 <template data-action="setFilter" data-common="filter"></template>
 
-<template data-action="createGeometry" data-common="map,filter">
+<template data-action="createGeometry" data-common="map,filter,setCustomValidity">
```

Comment:
It's a separate method, so it should also have it's own action in the actions dropdown instead of being included in the the create/editGeometry actions.

Reply:
Yes, correct :+1: